mlrun/api/utils/clients/chief.py

```diff
@@ -82,24 +83,42 @@ def invoke_schedule(
             "POST", f"projects/{project}/schedules/{name}/invoke", request
         )
 
-    def submit_job(self, request: fastapi.Request, body: dict) -> fastapi.Response:
-        return self._proxy_request_to_chief("POST", "submit_job", request, body)
+    def submit_job(self, request: fastapi.Request, json: dict) -> fastapi.Response:
+        return self._proxy_request_to_chief("POST", "submit_job", request, json)
 
-    def build_function(self, request: fastapi.Request, body: dict) -> fastapi.Response:
-        return self._proxy_request_to_chief("POST", "build/function", request, body)
+    def build_function(self, request: fastapi.Request, json: dict) -> fastapi.Response:
+        return self._proxy_request_to_chief("POST", "build/function", request, json)
 
     def delete_project(self, name, request: fastapi.Request) -> fastapi.Response:
         return self._proxy_request_to_chief("DELETE", f"projects/{name}", request)
 
+    def get_clusterization_spec(
+        self, proxy: bool = True, raise_on_failure: bool = False
```

Comment:
I don't think proxy is clear, I would go with something more specific for now - `return_fastapi_response`